app/views/links/_table.html.erb

```diff
@@ -31,7 +32,22 @@
               <td>
                 <%= link.description %>
               </td>
-              <td></td>
+              <% if action_buttons %>
```

Comment:
Ha nem rontja el a styling-ot, akkor én ezt az `if` -et a `<td></td>` -n belülre tenném. Szerintem olvashatóbb lenne, mert a `<td>` -t mindenképp meg akarom jeleníteni, csak a tartalma kérdéses.

Vagy ha ezt az oszlopot egyáltalán nem kellene megjelenítenünk, amikor az `action_buttons` hamis, akkor kerüljön a `table-head` -ben lévő `th` is egy `if` -be, és a jelenlegi `if` -nek pedig szűntessük meg az `else` ágát.